execute/plugin_functions.go

```diff
@@ -460,8 +460,10 @@ func computeMessageObservationsConsensus(
 
 				if msgsWithConsensus[0].IsPseudoDeleted() && !msgsWithConsensus[1].IsPseudoDeleted() {
 					msg = &msgsWithConsensus[1]
+					lggr.Infow("more than one msg reached consensus, selected second")
```

Comment:
```suggestion
					lggr.Infow("2 messages reached consensus with one being pseudo deleted. choosing full message")
```

same for the other one, saying we selected first or second doesn't give any useful info when just reading the logs without the code.